src/custom-lists/selectors.js

```diff
@@ -0,0 +1,104 @@
+import { createSelector } from 'reselect'
+
+import { selectors as filters } from 'src/overview/filters'
+
+// TODO: Needs some work.
+const sortAlphabetically = (a, b) => {
+    if (
+        typeof a === 'string' &&
+        typeof b === 'string' &&
+        a.name.toLowerCase() < b.name.toLowerCase()
+    )
+        return -1
+    if (
+        typeof a === 'string' &&
+        typeof b === 'string' &&
+        a.name.toLowerCase() > b.name.toLowerCase()
+    )
+        return 1
+    return 0
+}
+
+export const customLists = state => state.customLists
+
+export const fetchAllLists = createSelector(customLists, state => state.lists)
+export const getListCount = createSelector(
+    customLists,
+    state => state.listCount,
+)
+export const activeListIndex = createSelector(
+    customLists,
+    state => state.activeListIndex,
+)
+
+export const listFilterIndex = createSelector(
+    customLists,
+    state => state.listFilterIndex,
+)
+
+export const getSortedLists = createSelector(fetchAllLists, lists => {
```

Comment:
In the rest of the codebase, we have been naming our selector functions _without_ preceding verbs that most standard functions have. So instead of `getSortedLists` we would have called it simply `sortedLists`. Not really sure what's best long-term but, to be consistent with the rest of the code, I would encourage dropping those verbs.